lib/queries/asset-list.ts

```diff
@@ -14,3 +15,13 @@ export const queryAssetList = async (): Promise<AssetList> => {
 
   return await res.json();
 };
+
+// As this is an expensive operation, I think it's a good idea to cache it
+export const queryAssetFromAssetList = unstable_cache(
```

Comment:
It's a good idea but don't forget to add a revalidate prop or this cache will never expire

Reply:
We can use a long cache ofc

Reply:
https://github.com/osmosis-labs/osmosis-website/pull/70/commits/082f0cb08f91a3ea8769f8f4463084cf9e33080f